src/FpdiProtection.php

```diff
@@ -646,4 +687,31 @@ protected function _puttrailer()
             $this->_put('/ID [<' . $fileIdentifier . '><' . $fileIdentifier . '>]');
         }
     }
+
+    protected function getOpensslVersionNumber($patchAsNumber = false, $opensslVersionNumber = null) {
```

Comment:
This method looks like an overkill... there must be an easier way to check if OpenSSL 3 is in use.

Reply:
It isn't used at all anymore, great. So please remove it.